app/com/linkedin/drelephant/util/Utils.java

```diff
@@ -17,24 +17,33 @@
 package com.linkedin.drelephant.util;
 
 import com.linkedin.drelephant.analysis.Severity;
+import com.linkedin.drelephant.math.Statistics;
 import java.io.IOException;
 import java.io.InputStream;
+import java.text.DecimalFormat;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineManager;
 import javax.script.ScriptException;
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import javax.xml.parsers.ParserConfigurationException;
 import org.apache.hadoop.conf.Configuration;
+import models.AppResult;
+import org.apache.commons.io.FileUtils;
 import org.apache.log4j.Logger;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 import org.w3c.dom.Node;
 import org.w3c.dom.NodeList;
 import org.xml.sax.SAXException;
 import play.Play;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.*;
```

Comment:
As far as I know the convention we have been using is directly importing stuff, not using *s.